### ISSUE TITLE
Make the Donate button and page more prominent...

### ISSUE THREAD

**Issue by fredstluka** [RealRaven2000/QuickFolders#44]
Axel, as I mentioned in my long 28-point email 6/9/2020 2:57pm,
you should make the Donate button and Donate page more prominent.
They're too easy to miss, even when someone is actively looking
for them:

> 14. Just tripped across the Donate page.  I was looking for it
>     before but ended up just buying Pro for a year instead.
>     Now, at the Donate page, I was able to donate the $100
>     I had in mind, not just the $9/year subscription fee.
>     I found it by clicking on "You're brilliant. Can I reward you?"
>     in the FAQ at:
>     - http://quickfolders.org/faq_tips.html
>     which I got to from the QuickHelp tab of the Options tab
>     while I was trying to figure out a Reading List is.  
> 
>     You should make the Donate button/page more prominent.  I gave 
>     up on donating and just bought the Pro version, then later 
>     found the Donate page.

See my description at:
- https://github.com/RealRaven2000/QuickFolders/issues/44

of how I found the Premium Features page from a link on the 
Donate page:
- http://quickfolders.org/donate.html

I clicked the link and bought the Pro version.  It wasn't until much
later that I happened to go back to the Donate page and scrolled 
down far enough to discover the possibility to donating more than 
the mere $9 cost of the Pro version.

The Donate info about PayPal, etc. should be easier to find.  Even
though I was looking for it, I didn't see it on the Donate page.

Also, finding the Donate page itself was not easy.  I started out
wanting to donate $100.  I was already using QF for about a year.
Here's how I finally found the Donate page:
1. Clicked the QF cog at the top left for a dropdown menu
2. Clicked QuickFolder Options... to see Options dialog
3. Clicked Quick Help tab
4. Clicked "Find detailed information at the QuickFolders homepage at
  quickfolders.org..." to go to http://quickfolders.org
5. Clicked "FAQ + Tips" tab to go to http://quickfolders.org/faq_tips.html
6. Clicked "You're brilliant. Can I reward you?" to see its answer
7. Clicked "Just click here for detail." to see the Donate page

Along the way, I tried lots of other clicks, read lots of text, 
etc.  It took a LONG time to find the Donate page.  And even then
I ended up buying getting diverted and bought a $9 Pro license 
instead of making a $100 donation.  Doh!

QF version 4.18.1
TB version 68.9.0
Mac OS X version 10.11.6
--Fred

**Comment by RealRaven2000** [comment]
The main thing about "hiding" the donate page that it is much harder to tax donations. So I am generally more encouraging for people to buy an annual license as it is (ideally) a repeating fund (sustainable). I was also thinking along the lines of Patreon or similar...

I need to change some details to make donations easier & more official, so let's keep this one open and maybe discuss directly what to do...

**Comment by fredstluka** [comment]
> The main thing about "hiding" the donate page that it is much harder to tax donations. 

Hadn't thought of that.  Where are you located?  Ireland, right?  Do you have to collect sales tax and turn it in to the government or something?  In the US, it's kind a mess because each of the 50 states has it's own sales tax rate, policies, procedures, etc.


> So I am generally more encouraging for people to buy an annual license as it is (ideally) a repeating fund (sustainable). 

Yeah, recurring revenue is a good thing.

> I was also thinking along the lines of Patreon or similar...

Patreon is a platform for artists to sell their art, right?  How does it deal with taxes?

> I need to change some details to make donations easier & more official, so let's keep this one open and maybe discuss directly what to do...

OK.

--Fred

**Comment by fredstluka** [comment]
Oops!  Accidentally bumped "Close and comment".  Now reopened.

--Fred

**Comment by RealRaven2000** [comment]
> > The main thing about "hiding" the donate page that it is much harder to tax donations.
> 
> Hadn't thought of that. Where are you located? Ireland, right? Do you have to collect sales tax and turn it in to the government or something? In the US, it's kind a mess because each of the 50 states has its own sales tax rate, policies, procedures, etc.
> 

yes - so the way I do it with licenses: I don't . My shop (fastspring.com) does this for me, depending on in which geographical area the buyer sits, and just adds VAT / sales tax accordingly - nice. No such service with donations, just paypal taking a big cut (e.g. 7% of 100$, and probably 20% if it's one dollar).

> > I was also thinking along the lines of Patreon or similar...
> 
> Patreon is a platform for artists to sell their art, right? How does it deal with taxes?

I think technically patronage / donations are just "income" so one would add it to ones tax returns that way. 

If you want to donate to it to the paypal address axel@songcraftstudios.com adding the purpose (e..g quickFilters support donation)

**Comment by RealRaven2000** [comment]
I finally followed that advice after investing a lot of $$$ to get developer help for the Thunderbird 78 version. although i am not a great fan of popup windows this one was askew successful - it also offered extending existing licenses. with the side effect that a lot of users are now covered into 2022 which also means i will have less earnings next year (and am pushed towards registering for VAT for this year). so that may actually be a problem unless i find new users prepared to fund my projects long term.
Donations were quite successful this release because people probably were shocked that the conversion took so long.

**Comment by fredstluka** [comment]
Interesting...  Good luck with the financial aspects.  I hope you continue to get enough revenue to be able to continue.  I find QuickFolders VERY useful.  I donated $100 in 2020.  Maybe I'll donate again in 2021.  Occasional popup reminders asking for money may be a good idea.  That's what I see at WikiPedia, Free Software Foundation, Internet Archive, etc. 

--Fred

**Comment by crablouis** [comment]
I wouldn't find, say, quarterly donation nags to be annoying, with or without Pro License.
